data/events/scripts/player.lua

```diff
@@ -86,6 +86,25 @@ function Player:onLookInShop(itemType, count)
 end
 
 function Player:onMoveItem(item, count, fromPosition, toPosition, fromCylinder, toCylinder)
+	-- Store Inbox
```

Comment:
Wrong, events was created so we had a place to put the "old" C++ code when we moved it.
Idk why ppl seem to think that we should have everything in C++ when we can place it in Lua.

So IMO nothing wrong there.